### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## This Pr consists of 

- [ ] Created a new app. Give it a meaningful name.

- [ ] Initialized project with Git.

- [ ] Made sure that the project has a Postgres database set up

- [ ] Build my project schema.

- [ ] Created and run the necessary migration files.

### CODE REVIEW

File: db/migrate/20230125194631_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.string :comments_counter
+      t.string :likes_counter
```
Comment (AmaduKamara) [inline db/migrate/20230125194631_create_posts.rb]: - [ ] The type for comments_counter and likes_counter is expected to be an integer as there will be an increment operation that will happen as a user comments on a post or likes a post. Kindly work on these model types 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Kaweesi-Matia,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽 
✅ Good descriptive PR 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: db/migrate/20230125194631_create_posts.rb (hunk 1 above)
Comment (Kaweesi-Matia) [reply db/migrate/20230125194631_create_posts.rb]: changed data type of comments_counter andlikes_counter to integer

File: db/migrate/20230125185751_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :photo
+      t.text :Bio
+      t.integer :PostsCounter
```
Comment (Reem-lab) [inline db/migrate/20230125185751_create_users.rb:7]: - [ OPTIONAL ] I highly recommend to make the name here posts_counter to follow the best practices in name convensions so you can add a new migration and rename this column to be posts_counter here ✅😊🔥♻💯

**Review by Reem-lab — APPROVED** [review]
# Hi @Kaweesi-Matia, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Followed Gitflow ✅
- Descriptive PR ✅
- Great working on the functionlaity ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**